lua/ulx/modules/sh/cfc_cleanup.lua

```diff
@@ -5,41 +5,41 @@ local function ropeClean( callingPlayer, targetPlayers )
     for _, ply in pairs( targetPlayers ) do
         plyCounts[ply] = 0
     end
-    
+
     local ropes = ents.FindByClass( "keyframe_rope" )
     local ropeCount = 0
-    
+
     for _, rope in pairs( ropes ) do
         local owner = rope:CPPIGetOwner()
-        
-        if plyCounts[owner] ~= nil then 
+
+        if plyCounts[owner] ~= nil then
             rope:Remove()
             ropeCount = ropeCount + 1
             plyCounts[owner] = plyCounts[owner] + 1
         end
     end
 
-    ulx.fancyLogAdmin( callingPlayer, "#A removed "..ropeCount.." ropes owned by #T", targetPlayers )
+    ulx.fancyLogAdmin( callingPlayer, "#A removed " .. ropeCount .. " ropes owned by #T", targetPlayers )
 
     for ply, count in pairs( plyCounts ) do
         if count > 0 then
-            callingPlayer:ChatPrint( string.format( "%s owned %u ropes", ply:GetName(), count ) ) 
+            callingPlayer:ChatPrint( string.format( "%s owned %u ropes", ply:GetName(), count ) )
         end
     end
 end
 
 local ropes = ulx.command( CATEGORY_NAME, "ulx ropeclean", ropeClean, "!ropeclean" )
-ropes:addParam{ type=ULib.cmds.PlayersArg }
+ropes:addParam{ type = ULib.cmds.PlayersArg }
 ropes:defaultAccess( ULib.ACCESS_ADMIN )
-ropes:help( "Remove target(s) ropes" )
+ropes:help( "Remove target( s ) ropes" )
```

Comment:
should there be spacing in this string?
```suggestion
ropes:help( "Remove target(s) ropes" )
```

Reply:
no, that wasn't the intention